### DISCUSSION TITLE
How do I configure in Vim?

### DISCUSSION THREAD

**Discussion by legecha** [catppuccin/nvim#391] [Q&A] [answered]
I have installed in vanilla Vim:

```
Plug 'catppuccin/nvim', { 'as': 'catppuccin' }
```

I love the theme, but I want to remove the background. Is it possible to configure like so using vanilla Vim:

```
require("catppuccin").setup({
    transparent_background = false
})
```

If so, where should I put the above? And where/home would I call it? I kept reading about init.vim/init.lua but to my knowledge these are nvim things, not plain old Vim!

Thanks a bunch.

**Comment by rewhile** [comment, marked as answer]
In `.vimrc`:

```vim
lua << EOF
require("catppuccin").setup({
    transparent_background = false
})
EOF

colorscheme catppuccin-mocha
```

You can take a look at my [.vimrc](https://github.com/nullchilly/vimrc/blob/master/.vimrc) :)

**Reply by legecha** [reply]
Awesome, I've done that in my `.vimrc` and it works a treat. Thank you so much for the quick reply!
